flexible_event_config.md

```diff
@@ -33,6 +33,9 @@ This proposal is broken into two separate feature sets:
 Phase 2 (Full flexible event-level) could be used to do all of the capabilities in Phase 1 and:
 * Vary the trigger data cardinality in a report
 * Reduce the amount of total noise by decreasing the trigger data cardinality
+* Implemented [as described in the primary explainer](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#data-limits-and-noise).
```

Comment:
I find this bulletpoint confusing, as it makes it sound like "Full flexible event-level" has been implemented.

Reply:
+1, should we just take summary buckets/multiple specs out of phase 2 and mark as ideas for future improvements?

Reply:
Thanks for feedback. Just edited the explainer to make this more clear